conf/config.py

```diff
@@ -173,6 +174,11 @@ def _set_config(self, request_type, key, value, option="update"):
             res["addition"] = tmp
 
             self.conf_data[key] = value
+            if value in self.conf_data.keys():
+                self.conf_data[value] = description.DefaultValue.get_defaultvalue_by_key(value)
+            if len(res["addition"]) != 0:
+                if value == res["addition"][0]['key']:
```

Comment:
Hi, I don't understand what this part is for? Seems you are planing to fill the value of additional keys, but why compare value with addidtion[0]['key']?